src/opnsense/mvc/app/views/OPNsense/Diagnostics/health.volt

```diff
@@ -1,623 +1,187 @@
-                        </table>
-                    </div>
-                </div>
-            </div>
+        <div class="label-select-pair">
+            <label for="health-category-select"><b>{{ lang._('Category') }}</b></label>
+            <select id="health-category-select" class="selectpicker" data-width="200px" data-container="body"></select>
+        </div>
+        <div class="label-select-pair">
+            <label for="health-subcategory-select"><b>{{ lang._('Subject') }}</b></label>
+            <select id="health-subcategory-select" class="selectpicker" data-width="200px" data-live-search="true" data-container="body"></select>
         </div>
-    </div>
 
-    <div id="chart_details_table" class="col-md-12" style="display: none;">
-        <div class="panel panel-default">
-            <div class="panel-heading">
-                <h3 class="panel-title">
-                  {{ lang._('Current View - Details') }}
-                </h3>
-            </div>
-            <div class="panel-body">
-                <div class="btn-toolbar" role="toolbar">
-                    <i>{{ lang._('Toggle Timeview') }}:</i>
-
-                    <div class="btn-group update_options" data-toggle="buttons">
-                        <label class="btn btn-xs btn-default active">
-                            <input type="radio" id="time0" name="toggle_time" checked="checked" value="0"/> {{
-                            lang._('Timestamp') }}
-                        </label>
-                        <label class="btn btn-xs btn-default">
-                            <input type="radio" id="time1" name="toggle_time" value="1"/> {{ lang._('Full Date & Time') }}
-                        </label>
-                    </div>
-                    <div class="btn btn-xs btn-primary inline" onclick='downloadCSV({ filename: rrd+".csv" });'>
-                        <i class="fa fa-download"></i> {{ lang._('Download as CSV') }}
-                    </div>
-                </div>
-
-                <div class="table-responsive">
-                    <table class="table table-condensed table-hover table-striped">
-                        <thead>
-                        <tr id="table_view_heading" class="active">
-                            <!-- Dynamic data -->
-                        </tr>
-                        </thead>
-                        <tbody id="table_view_rows">
-                        <!-- Dynamic data -->
-                        </tbody>
-                    </table>
-                </div>
-            </div>
+        <div class="label-select-pair ">
+            <label for="detail-select"><b>{{ lang._('Granularity') }}</b></label>
+            <select id="detail-select" class="selectpicker" data-width="200px">
+                <option value="0">Default (1 minute)</option>
+                <option value="1">5 minutes</option>
+                <option value="2">1 hour</option>
+                <option value="3">24 hours</option>
```

Comment:
missing translations, the step size is more or less dynamic, which is what the following block tried to figure out:

https://github.com/opnsense/core/blob/464bd59ec3c7a3731b8bc95f43b4ac0d6e48bf74/src/opnsense/mvc/app/views/OPNsense/Diagnostics/health.volt#L233-L253